digiwf-integrations/digiwf-address-integration/README.md

```diff
@@ -1,12 +1,83 @@
-# Digiwf Address Integration 
+# DigiWF Address Integration
 
-The following steps are needed to run the integration locally.
+The Address Integration connects the address service of the City of Munich to the DigiWF platform, allowing information
+about addresses and streets to be retrieved.
+
+## Usage of the Spring Boot Starter
+
+The DigiWF Address Integration is provided as a Spring Boot Starter project. It was implemented in a hexagonal
+architecture to ensure adaptability and extensibility.
+
+You can integrate the `digiwf-address-integration-starter` into your project as follows:
+
+**With Maven**
+
+```xml
+
+<dependency>
+    <groupId>de.muenchen.oss.digiwf</groupId>
+    <artifactId>digiwf-address-integration-starter</artifactId>
+    <version>${digiwf.version}</version>
+</dependency>
+```
+
+**With Gradle**
+
+```gradle
+implementation group: 'de.muenchen.oss.digiwf', name: 'digiwf-address-integration-starter', version: '${digiwf.version}'
+```
+
+To extend or replace the functions of the integration, you only need to override the port interfaces and provide them
+as `@Bean`. This will replace our standard implementation with your custom implementation.
+
+You can find the port definitions at the
+path: [digiwf-address-integration-core/src/main/java/de/muenchen/oss/digiwf/address/integration/application/port](https://github.com/it-at-m/digiwf-core/tree/dev/digiwf-integrations/digiwf-address-integration/digiwf-address-integration-core/src/main/java/de/muenchen/oss/digiwf/address/integration/application/port).
+
+The Address Client is responsible for communication with the City of Munich's Address Service. The client was
+implemented as a library in its own Maven module to encapsulate dependencies. The implementation of the client can also
+be extended or replaced by implementing the API interfaces AddressGermanyApi, AddressMunichApi, and StreetsMunichApi.
+These interfaces must also be provided as `@Bean`.
 
-## Getting started
+You can find the API interfaces at the
+path: [digiwf-address-integration-client/src/main/java/de/muenchen/oss/digiwf/address/integration/client/api](https://github.com/it-at-m/digiwf-core/tree/dev/digiwf-integrations/digiwf-address-integration/digiwf-address-integration-client/src/main/java/de/muenchen/oss/digiwf/address/integration/client/api)
```

Comment:
Das mit den Startern würde ich hier eigentlich nicht erklären, nachdem das ja für jede Integration gleich ist. Evtl. macht es Sinn eine README direkt in den digiwf-integrations-Ordner zu legen in der die Architektur, Paket-Benennung und welches Paket was macht beschrieben wird.